index.html

```diff
@@ -64,63 +70,85 @@ <h1 class="item1 h1">I’m Kirubel. Glad to see you!</h1>
       </div>
       <button>See Project</button>
     </ul>
-    <img src="./images/Snapshoot.png" alt="snapshot of website">
-    <p class="oldo">Multi-Post<br>Stories</p>
+    </div>
+  </div>
+  <div class="back">
+    <div class="bodydiv bodydiv2">
+    <p class="oldo">Multi-Post Stories</p>
     <ul>
       <div class="work1-li1">
-        <li class="work1-item1 work1-item3">CANOPAY</li>
-        <li class="work1-item1 work1-item2">Back End Dev</li>
+        <li class="work1-item1 work1-item3">FACEBOOK</li>
+        <li class="work1-item1 work1-item2">Full Stack End Dev</li>
         <li class="work1-item1 work1-item2">2015</li>
       </div>
       </ul>
-       <p class="pp">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+       <p class="pp">Experimental content creation feature that allows users to add 
+        to an existing story over the course of a day without spamming their friends.</p>
        <ul>
        <div class="work1-link">
       <li >HTML</li>
+      <li class="newlin">Ruby on rails</li>
       <li class="newlin">CSS</li>
       <li class="newlin">JavaScript</li>
       </div>
     </ul>
       <button>See Project</button>
-      <section class="work1">
-        <img class="work1-img" src="./images/Snapshoot.png" alt="snapshot of website">
-        <p class="oldo">Tonic</p>
+    </div>
+    <img class="work1-img bodydiv" src="./images/port.png" alt="snapshot of website">
+  </div>
+  <div class="back">
+    <div>
+        <img class="work1-img bodydiv" src="./images/Snapshoot Portfolio (1).png" alt="snapshot of website">
+      </div>
+        <div class="bodydiv bodydiv3">
+        <p class="oldo">Facebook 360</p>
```

Comment:
Your design at screen size (863px)
![image](https://user-images.githubusercontent.com/78251927/229249349-5526ea0a-484a-43e4-a0f2-7b5e69c24bc7.png)

- [ ] I notice that the page is not responsive and I can see a horizontal scroll bar when I resize the screen🤔

- [ ] Please let's fix the issues with the page responsiveness, It is important that our design remains responsive in the desktop version(768px or higher).

- [ ] Kindly use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px).